tests/adapters/compliance_tests/unset_interface_state_test.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2015 Internap.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from hamcrest import assert_that, is_
+from netman.core.objects.exceptions import UnknownInterface
+
+from tests.adapters.configured_test_case import ConfiguredTestCase
+
+
+class UnsetInterfaceStateTest(ConfiguredTestCase):
+    _dev_sample = "juniper"
+
+    def test_unsetting_an_interface(self):
+        self.try_to.unset_interface_state(self.test_ports[0].name)
```

Comment:
you're trying to unset and interface state after not setting it?

i also don't really see why you need 2 interfaces.

Wouldn't this test work out:

original_state = get state
set state ON
unset
assert new state == original
set state OFF
unset
assertt new state == original